app/views/ChargeSummary.scala.html

```diff
@@ -312,24 +312,24 @@ <h2 class="govuk-heading-l">@messages("chargeSummary.chargeHistory.heading")</h2
                         }
                         </td>
                         <td class="govuk-table__cell numeric govuk-!-text-align-right">@{
-                            documentDetail.originalAmount.getOrElse(throw MissingFieldException("Original Amount")).toCurrency
+                            documentDetail.originalAmount.get.toCurrency
```

Comment:
I think this is a way to hide problem rather than resolve issue;
If this can be address with the right view model pls

Reply:
agree, a view model is the right solution as it has different constraints to the domain model for example

Reply:
having said that it may require a wider architecture standards review so that we have a consistent approach in development and a consistent code review process

Reply:
I was under strong impression that this in scope of this story, as we already done investigation how to approach this type of stories; but yes, really up to you how this would be delivered